src/app/components/resource/resource.service.js

```diff
@@ -103,4 +104,34 @@ export class ResourceService {
   }
 
 
+  editUser(fullname, username, password, email, regno1, regno2) {
+    if (regno1 === undefined) {
```

Comment:
Just a suggestion:

use multi input for plate number since there is no real limitation how many numbers user can have, why we should decide that 2 is the limit?

How multi input could look like: https://bootstrap-tagsinput.github.io/bootstrap-tagsinput/examples/